posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -200,9 +124,6 @@ def where(self):
 
         return ast.And(exprs=exprs)
 
-    def group_by(self):
```

Comment:
We never want to group_by issue_id? I kinda figured that'd be the default, when we're not looking for a specific id, and I think the `min` and `max` operations in the `select` imply it?

Reply:
We should always group by `issue_id`! Moved it out of a function because we always do it now (see the select statement)

Reply:
Ah, I see it in `to_query`, missed on my first pass through 🤦